src/Stormpath/Client.php

```diff
@@ -106,7 +108,9 @@ public function __construct(ApiKey $apiKey, $cacheManager, $cacheManagerOptions,
         self::$cacheManager = $cacheManager;
         self::$cacheManagerOptions = $cacheManagerOptions;
 
-        $requestExecutor = new HttpClientRequestExecutor();
+        $signer = $this->resolveSigner();
+        $requestExecutor = new HttpClientRequestExecutor(new $signer);
```

Comment:
These two lines shouldn't be necessary as the `HttpClientRequestExecutor ` should default to the Sauthc1 signer if no argument is passed. The `ClientBuilder` should set the signer if an authentication scheme was provided, after creating the client.